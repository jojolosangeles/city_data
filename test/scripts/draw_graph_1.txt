load expected/all_areas.csv as all_areas
filter all_areas by Area Name
mbar all_areas Area Name X|Year:O|Year Y|count()|Number+of+Accidents
