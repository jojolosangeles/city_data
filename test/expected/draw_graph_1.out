Test: scripts/draw_graph_1.txt
> load expected/all_areas.csv as all_areas
> filter all_areas by Area Name
Execute FilterByCommand
all_areas => Area Name
Register UNIQUE VALUES FOR Area Name
['Central' '77th Street' 'Pacific' 'Topanga' 'Southwest' 'Foothill'
 'Newton' 'Hollywood' 'N Hollywood' 'Northeast' 'Hollenbeck' 'West LA'
 'Southeast' 'Devonshire' 'Mission' 'Harbor' 'Wilshire' 'Rampart'
 'Olympic' 'West Valley' 'Van Nuys']
FILTERED to all_areas.Central
put_data_frame(all_areas.Central)
FILTERED to all_areas.77th_Street
put_data_frame(all_areas.77th_Street)
FILTERED to all_areas.Pacific
put_data_frame(all_areas.Pacific)
FILTERED to all_areas.Topanga
put_data_frame(all_areas.Topanga)
FILTERED to all_areas.Southwest
put_data_frame(all_areas.Southwest)
FILTERED to all_areas.Foothill
put_data_frame(all_areas.Foothill)
FILTERED to all_areas.Newton
put_data_frame(all_areas.Newton)
FILTERED to all_areas.Hollywood
put_data_frame(all_areas.Hollywood)
FILTERED to all_areas.N_Hollywood
put_data_frame(all_areas.N_Hollywood)
FILTERED to all_areas.Northeast
put_data_frame(all_areas.Northeast)
FILTERED to all_areas.Hollenbeck
put_data_frame(all_areas.Hollenbeck)
FILTERED to all_areas.West_LA
put_data_frame(all_areas.West_LA)
FILTERED to all_areas.Southeast
put_data_frame(all_areas.Southeast)
FILTERED to all_areas.Devonshire
put_data_frame(all_areas.Devonshire)
FILTERED to all_areas.Mission
put_data_frame(all_areas.Mission)
FILTERED to all_areas.Harbor
put_data_frame(all_areas.Harbor)
FILTERED to all_areas.Wilshire
put_data_frame(all_areas.Wilshire)
FILTERED to all_areas.Rampart
put_data_frame(all_areas.Rampart)
FILTERED to all_areas.Olympic
put_data_frame(all_areas.Olympic)
FILTERED to all_areas.West_Valley
put_data_frame(all_areas.West_Valley)
FILTERED to all_areas.Van_Nuys
put_data_frame(all_areas.Van_Nuys)
> mbar all_areas Area Name X|Year:O|Year Y|count()|Number+of+Accidents
Column Descriminator = 'Area Name'
BarGraphCommand(all_areas) ['X', 'Year:O', 'Year'], ['Y', 'count()', 'Number of Accidents']
REMEMBER UNIQUE VALUES FOR Area Name
Unique Values For Column = ['Central' '77th Street' 'Pacific' 'Topanga' 'Southwest' 'Foothill'
 'Newton' 'Hollywood' 'N Hollywood' 'Northeast' 'Hollenbeck' 'West LA'
 'Southeast' 'Devonshire' 'Mission' 'Harbor' 'Wilshire' 'Rampart'
 'Olympic' 'West Valley' 'Van Nuys']
VALUE=Central
BarGraphCommand(Central) ['X', 'Year:O', 'Year'], ['Y', 'count()', 'Number of Accidents']
Saved Central.png
VALUE=77th Street
BarGraphCommand(77th_Street) ['X', 'Year:O', 'Year'], ['Y', 'count()', 'Number of Accidents']
Saved 77th_Street.png
VALUE=Pacific
BarGraphCommand(Pacific) ['X', 'Year:O', 'Year'], ['Y', 'count()', 'Number of Accidents']
Saved Pacific.png
VALUE=Topanga
BarGraphCommand(Topanga) ['X', 'Year:O', 'Year'], ['Y', 'count()', 'Number of Accidents']
Saved Topanga.png
VALUE=Southwest
BarGraphCommand(Southwest) ['X', 'Year:O', 'Year'], ['Y', 'count()', 'Number of Accidents']
Saved Southwest.png
VALUE=Foothill
BarGraphCommand(Foothill) ['X', 'Year:O', 'Year'], ['Y', 'count()', 'Number of Accidents']
Saved Foothill.png
VALUE=Newton
BarGraphCommand(Newton) ['X', 'Year:O', 'Year'], ['Y', 'count()', 'Number of Accidents']
Saved Newton.png
VALUE=Hollywood
BarGraphCommand(Hollywood) ['X', 'Year:O', 'Year'], ['Y', 'count()', 'Number of Accidents']
Saved Hollywood.png
VALUE=N Hollywood
BarGraphCommand(N_Hollywood) ['X', 'Year:O', 'Year'], ['Y', 'count()', 'Number of Accidents']
Saved N_Hollywood.png
VALUE=Northeast
BarGraphCommand(Northeast) ['X', 'Year:O', 'Year'], ['Y', 'count()', 'Number of Accidents']
Saved Northeast.png
VALUE=Hollenbeck
BarGraphCommand(Hollenbeck) ['X', 'Year:O', 'Year'], ['Y', 'count()', 'Number of Accidents']
Saved Hollenbeck.png
VALUE=West LA
BarGraphCommand(West_LA) ['X', 'Year:O', 'Year'], ['Y', 'count()', 'Number of Accidents']
Saved West_LA.png
VALUE=Southeast
BarGraphCommand(Southeast) ['X', 'Year:O', 'Year'], ['Y', 'count()', 'Number of Accidents']
Saved Southeast.png
VALUE=Devonshire
BarGraphCommand(Devonshire) ['X', 'Year:O', 'Year'], ['Y', 'count()', 'Number of Accidents']
Saved Devonshire.png
VALUE=Mission
BarGraphCommand(Mission) ['X', 'Year:O', 'Year'], ['Y', 'count()', 'Number of Accidents']
Saved Mission.png
VALUE=Harbor
BarGraphCommand(Harbor) ['X', 'Year:O', 'Year'], ['Y', 'count()', 'Number of Accidents']
Saved Harbor.png
VALUE=Wilshire
BarGraphCommand(Wilshire) ['X', 'Year:O', 'Year'], ['Y', 'count()', 'Number of Accidents']
Saved Wilshire.png
VALUE=Rampart
BarGraphCommand(Rampart) ['X', 'Year:O', 'Year'], ['Y', 'count()', 'Number of Accidents']
Saved Rampart.png
VALUE=Olympic
BarGraphCommand(Olympic) ['X', 'Year:O', 'Year'], ['Y', 'count()', 'Number of Accidents']
Saved Olympic.png
VALUE=West Valley
BarGraphCommand(West_Valley) ['X', 'Year:O', 'Year'], ['Y', 'count()', 'Number of Accidents']
Saved West_Valley.png
VALUE=Van Nuys
BarGraphCommand(Van_Nuys) ['X', 'Year:O', 'Year'], ['Y', 'count()', 'Number of Accidents']
Saved Van_Nuys.png
Test completed: scripts/draw_graph_1.txt
